### DISCUSSION TITLE
Welcome to baker Discussions!

### DISCUSSION THREAD

**Discussion by zhenkewu** [zhenkewu/baker#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jkubale** [comment]
Hi, I've been working through the vignette (reproduce_baker_software.html) and accompanying Rmarkdown document (reproduce_baker_software.rmd) and have successfully run the 3 example models. However, when I get to the section "Visualizations (NPLCM with regression)" I get the following error message:

`Error in compute_marg_PR_nested_reg(set_parameter$ThetaBS[1:N.CAUSE, ],  : 
  could not find function "compute_marg_PR_nested_reg"`

I've looked through the code in the vignette and the Rmd and can't seem to find where this function is defined. Any help in resolving this would be greatly appreciated!

**Reply by zhenkewu** [reply]
`compute_marg_PR_nested_reg` is an internal function not intended to be called in Global environment (when R loads baker, it will not make this function explicitly available to call). But if yo really want to do this, then use "baker:::compute_marg_PR_nested_reg". If this is in the vignette, then we have to either export the "compute_marg_PR_nested_reg" function or use the ":::" operator @realirena

**Reply by zhenkewu** [reply]
The function is defined on line 178 here: https://github.com/zhenkewu/baker/blob/master/R/compute_functionals_of_model.R
